neo/Prompt/Commands/LoadSmartContract.py

```diff
@@ -235,5 +235,7 @@ def ImportMultiSigContractAddr(wallet, args):
         wallet.AddContract(verification_contract)
 
         print("Added multi-sig contract address %s to wallet" % address)
+        
+        return address
 
     return 'Hello'
```

Comment:
Might this `return 'Hello'` not be a problem for your use-case?

Reply:
In my use-case, I'll be interacting with the newly created multisig contract address. So having ImportMultiSigContractAddr() return this address would be helpful.